alvr/xtask/src/dependencies.rs

```diff
@@ -157,69 +202,65 @@ pub fn build_ffmpeg_linux(nvenc_flag: bool) {
            Nvidia docs:
            https://docs.nvidia.com/video-technologies/video-codec-sdk/ffmpeg-with-nvidia-gpu/#commonly-faced-issues-and-tips-to-resolve-them
         */
-        #[cfg(target_os = "linux")]
-        {
-            let codec_header_version = "12.1.14.0";
-            let temp_download_dir = download_path.join("dl_temp");
-            command::download_and_extract_zip(
-                &format!("https://github.com/FFmpeg/nv-codec-headers/archive/refs/tags/n{codec_header_version}.zip"),
-                &temp_download_dir
-            )
-            .unwrap();
+        let codec_header_version = "12.1.14.0";
```

Comment:
tl.dr didn't change anything there but remove wrap and cfg for linux, it's anyway ran only on linux

Reply:
nvm it's actually for a reason, windows builds that too